app/models/orphan.rb

```diff
@@ -127,6 +127,12 @@ def current_sponsor
 private
 
   def sponsored_siblings_does_not_exceed_siblings_count
+    if minor_siblings_count.nil?
+      unless sponsored_minor_siblings_count == 0 or sponsored_minor_siblings_count.nil?
+        errors.add(:sponsored_minor_siblings_count, "cannot exceed minor siblings count")
+      end
+      return
+    end
     if sponsored_minor_siblings_count && (sponsored_minor_siblings_count > minor_siblings_count)
       errors.add(:sponsored_minor_siblings_count, "cannot exceed minor siblings count")
     end
```

Comment:
what about the minor_sibling_count.nil? guard - thats the crucial thing. 
Given neither are compulsory now the minor siblings_count could be nil while the sponsored_minor_siblings_count could have a value